build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "3.2.5"
+    id("io.spring.dependency-management") version "1.1.4"
+    id("com.github.ben-manes.versions") version "0.51.0"
+    id("org.jmailen.kotlinter") version "4.3.0"
+    kotlin("jvm") version "1.9.22"
+    kotlin("plugin.spring") version "1.9.22"
+    id("net.thebugmc.gradle.sonatype-central-portal-publisher") version "1.2.3"
+    id("maven-publish")
+}
+
+group = "com.valensas"
+version = "0.0.0"
```

Comment:
Versiyonu buraya yazmak yerine pipeline'da `-Pversion=...` olarak tanımlayalım.

Reply:
Örnek: https://github.com/Valensas/observability/blob/master/.github/workflows/publish.yaml#L25

Reply:
Onun eklenmesine rağmen version olması gerekli değil mi dev süreci için, yoksa silelim mi

Reply:
Kaldırabilirsin, development için gerekli değil